test/e2e/framework/skipper/skipper.go

```diff
@@ -100,9 +100,14 @@ func SkipIfFeatureGateEnabled(gate featuregate.Feature) {
 	}
 }
 
-// SkipIfMissingResource skips if the gvr resource is missing.
-func SkipIfMissingResource(ctx context.Context, dynamicClient dynamic.Interface, gvr schema.GroupVersionResource, namespace string) {
-	resourceClient := dynamicClient.Resource(gvr).Namespace(namespace)
+// SkipIfMissingResource skips if the gvr resource is missing. If namespaced is false, namespace is ignored.
+func SkipIfMissingResource(ctx context.Context, dynamicClient dynamic.Interface, gvr schema.GroupVersionResource, namespaced bool, namespace string) {
+	var resourceClient dynamic.ResourceInterface
```

Comment:
Why do we need it now? What about some other existing global resources without namespace?

Reply:
we don't need it. once https://github.com/kubernetes/kubernetes/pull/121895 is merged, we can add a feature label to the spec. for example

```go
utils.SIGDescribe("VolumeAttributesClass", features.VolumeAttributesClass, func() {
```